packages/portalnetwork/src/client/eth.ts

```diff
@@ -112,6 +119,28 @@ export class ETH {
     }
     return block
   }
+
+  getBlockByTag = async (
+    blockTag: 'latest' | 'finalized',
+    includeTransactions: boolean,
+  ): Promise<Block | undefined> => {
+    // Requires beacon light client to be running to get `latest` or `finalized` blocks
+    this.networkCheck([NetworkId.BeaconChainNetwork])
+    let clHeader
+    switch (blockTag) {
+      case 'latest': {
+        clHeader = this.beacon!.lightClient?.getHead() as capella.LightClientHeader
```

Comment:
Not to resolve here but note to self to update the ssz typings.  We should be able to make this future proof if I can remember how to do it.